src/main/java/org/generationcp/commons/constant/DefaultGermplasmStudyBrowserPath.java

```diff
@@ -10,5 +10,6 @@
 	String GERMPLASM_BROWSER_LINK = "ibpworkbench/maingpsb/germplasm-";
```

Comment:
I still found references to this old germplasm browser link (GERMPLASM_BROWSER_LINK). Should those have been updated as well to use the new link?

Reply:
This is done.